README.md

```diff
@@ -73,141 +37,228 @@ line tool:
 
 [releases]: https://github.com/amonks/run/releases
 
-## Task Files
+## Quick Start
```

Comment:
This quick start doesn't actually work. i think there's more value in a quick start that runs and doesn't show all features than one that doesn't run. 🤷

Reply:
another con is that it uses dependencies incorrectly 😅 — should be a trigger

Reply:
first stab:

```
[[task]]
  id = "dev"
  type = "long"
  dependencies = ["build"]
  cmd = "npx serve dist"

[[task]]
  id = "build"
  type = "short"
  watch = ["*.html"]
  cmd = """
    mkdir -p dist
    cat header.html body.html footer.html > dist/index.html
  """
```

it still doesn't work as-written, but it might be legible to non-go-users. still not ideal, but maybe an improvement?

Reply:
hmm, maybe NPM scripts are sufficiently widely known to be a common-ground example?

```toml
[[task]]
  id = "dev"
  type = "long"
  triggers = ["build"]
  cmd = "npm start"

[[task]]
  id = "build"
  type = "short"
  watch = ["src/**/*.js"]
  cmd = "npm build"
```

I'm hoping a reader would see this and think "since run isn't a javascript project, `npm start` and `npm run` must be stand-ins for my own project scripts; not something run-specific. I see how I would insert my own scripts here when integrating run with my project"

I imagine that most readers will have some code project they are already working on — most people are not starting a new project today —  so I think "how do I integrate run into my existing project" is more salient than "how do I start a new project that uses run"

Reply:
maybe it is better to communicate this stuff explicitly, rather than hoping people get the right idea.

```toml
[[task]]
  id = "dev"
  type = "long"
  triggers = ["build"]
  # substitute whatever script starts your server
  cmd = "npm start"

[[task]]
  id = "build"
  type = "short"
  watch = ["src/**/*.js"]
  # substitute whatever script builds your project
  # this will run whenever src/**/*.js changes
  # after this runs, the "dev" task will restart too
  cmd = "npm build"
```

Reply:
Hmm, I think maybe quick start can be different things. When I open a repo for the first time and see `Quick Start` I think hello world, copy paste run and it's working. But I see what you mean about realistic being valuable for this case.

What if we had something like

"After installation, to start integrating `run` with your project create a root tasks.toml folder, and copy paste this task"

[[task]]
  id= "info"
  type = "short"
  env = {
    PROJECT_NAME: <your project>
  }
  cmd = """
    echo Welcome to the $PROJECT_NAME project!
  """
}

this would be functional and realistic-ish

Reply:
ooooh; great idea to use narrative to describe how to integrate

I think that leads to a best-of-all-worlds implementation which,
- describes narratively how to think about integrating run with a pre-existing project
- uses realistic tasks (eg "build" and "dev" and "test" rather than "info")

Reply:
I ended up breaking quick start into two sections: a working (but unrealistic) example, and an integration guide

Reply:
squashing the heck out of this branch now